app-ios/Modules/Sources/Timetable/TimetableListItemView.swift

```diff
@@ -14,16 +17,28 @@ struct TimetableListItemView: View {
         HStack(alignment: .top) {
             VStack(alignment: .leading, spacing: 0) {
                 Spacer().frame(height: 4)
-                Text(timetableItem.language.langOfSpeaker)
-                    .font(Font.system(size: 12, weight: .medium))
-                    .padding(.horizontal, 8)
-                    .padding(.vertical, 4)
-                    .overlay(
-                        RoundedRectangle(cornerRadius: 4)
-                            .inset(by: 0.5)
-                            .stroke(AssetColors.outline.swiftUIColor, lineWidth: 1)
+                // TODO apply like flexbox layout
+                LazyHStack(spacing: 4) {
+                    // TODO use AssertColors
+                    let labelColor: Color = if colorScheme == .light {
+                        .white
+                    } else {
+                        .black
+                    }
```

Comment:
How about this TODO? 👀

Reply:
Sorry, added! https://github.com/DroidKaigi/conference-app-2023/pull/724/commits/62bf81c63b2118b3964e176bfb61dc885a0daafa

@ry-itto 
Currently, I didn't add colors for dark mode. I think it is better to add them in other issue.
If it is acceptable for you, let me create the issue 🙏

Reply:
Addressing it in https://github.com/DroidKaigi/conference-app-2023/pull/777